Kernel/Devices/DeviceManagement.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2021, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/HashTable.h>
+#include <AK/Singleton.h>
+#include <Kernel/Devices/DeviceManagement.h>
+#include <Kernel/FileSystem/InodeMetadata.h>
+#include <Kernel/Sections.h>
+
+namespace Kernel {
+
+static Singleton<DeviceManagement> s_the;
+
+UNMAP_AFTER_INIT DeviceManagement::DeviceManagement()
+{
+}
+UNMAP_AFTER_INIT void DeviceManagement::initialize()
+{
+    s_the.ensure_instance();
+}
+
+UNMAP_AFTER_INIT void DeviceManagement::attach_null_device(Device const& device)
+{
+    VERIFY(device.major() == 1);
+    VERIFY(device.minor() == 3);
```

Comment:
Can we instead make this take a `NullDevice const&` and remove these VERIFYs? compile-time constraints are much nicer than runtime ones :)